File: logic-fizzbuzz/FizzBuzz.java
class FizzBuzz {
    public static void main (String[] args){
        for(int ii=0; ii <= 100; ii++){
            if (ii % 15 == 0) {
                System.out.println("FizzBuzz");
            } else if (ii % 3 == 0) {
                System.out.println("Fizz");
            } else if (ii % 5 == 0) {
                System.out.println("Buzz");
            } else {
                System.out.println(ii);
            }
        }
    }
}

File: logic-fizzbuzz/FizzBuzz.exs
(1..100) |> Enum.each(fn (ii) -> cond do
    rem(ii, 15) === 0 -> IO.puts "FizzBuzz"
    rem(ii, 3) === 0 -> IO.puts "Fizz"
    rem(ii, 5) === 0 -> IO.puts "Buzz"
    true -> IO.puts ii
  end
end)
